pyqir-jit/src/python.rs

```diff
@@ -73,7 +76,9 @@ impl PyNonadaptiveJit {
             Ok(())
         }
 
-        let gen_model = jit::run_module_file(file, entry_point).map_err(PyOSError::new_err)?;
+        let context = Context::create();
```

Comment:
I would prefer to put the `run_module_file` method back in. I don't think we should be surfacing inkwell and qirlib in the python interop layer.